share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/requierments.txt

```diff
@@ -0,0 +1,4 @@
+numpy
```

Comment:
Please correct the spelling of the file's name.

Reply:
done